doc/source/develop.rst

```diff
@@ -0,0 +1,79 @@
+Developing Streamparse
+======================
+
+Lein
+------
+
+Install Leiningen according to the instructions in the quickstart.
+
+
+Using Local Clojure Interop Library
+------------------------------------
+
+You can tell ``lein`` to point directly at streamparse's Clojure repo and use the
+code there for all of the interop commands, so that you can test changes while
+developing.
+
+To do this, add a directory called ``checkouts`` and symlink it up::
+
+    mkdir checkouts
+    cd checkouts
+    ln -s ../../../streamparse/jvm streamparse
+    cd ..
+
+Now, comment out the ``com.parsely/streamparse`` dependency in ``project.clj``. 
+It will now pick up the Clojure commands from your local repo. So, now you can
+tweak and change them!
+
+Local pip installation
+----------------------
+
+In your virtualenv for this project, go into ``~/repos/streamparse`` (where you 
+cloned streamparse) and simply run::
+
+    python setup.py develop
+
+This will install a streamparse Python version into the virtualenv which is
+essentially symlinked to your local version.
+
+**NOTE:** streamparse currently pip installs streamparse's **released** version
```

Comment:
It seems like there should be an issue for this, since you can probably get bitten pretty hard by this.